ontology/observable/observable.ttl

```diff
@@ -4511,6 +4511,12 @@ observable:OperatingSystemFacet
 	rdfs:label "OperatingSystemFacet"@en ;
 	rdfs:comment "An operating system facet is a grouping of characteristics unique to the software that manages computer hardware, software resources, and provides common services for computer programs. [based on https://en.wikipedia.org/wiki/Operating_system]"@en ;
 	sh:property
+		[
+			sh:class core:IdentityAbstraction ;
```

Comment:
core:IdentityAbstraction should only be used within the core namespace. It is a mechanism to allow references to Identity within core without causing cyclical importing.
In the observable namespace, uses such as this should directly use identity:Identity.

Reply:
Resolved in d7a593e.